packages/smart-contracts/src/lib/artifacts/ERC20EscrowToPay/index.ts

```diff
@@ -33,6 +33,11 @@ export const erc20EscrowToPayArtifact = new ContractArtifact<ERC20EscrowToPay>(
           address: '0x4BA012eae4d64da79Bd6bcdBa366803fCe701A4C',
           creationBlockNumber: 18086337,
         },
+        // FIXME: Address changed due to contract metadata change since last deployment
```

Comment:
I don't understand what there is to fix?

Reply:
Nothing to fix, just a comment to explain why the address is not the same as for other networks.

Reply:
then remove `FIXME`?